fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true
+    };
+  }
+
+  componentDidMount() {
+
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(data => this.setState({ isLoading: !this.state.isLoading, users: data }))
```

Comment:
you don't want to flip here, it should only set to true, because that means the data is done loading.